styles.css

```diff
@@ -123,3 +117,353 @@ nav {
+  line-height: 24px;
+  margin-top: 24px;
+  margin-bottom: 24px;
+  padding-right: 12px;
+}
+
+.get-resume-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: auto;
+  height: 48px;
+  background: #ff6b00;
+  border: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  margin-bottom: 209px;
+}
+
+.about-me hr {
+  border: 1px solid white;
+}
+
+.languages,
+.frameworks,
+.skills {
+  font-family: 'inter', sans-serif;
+  text-align: center;
+  list-style: none;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages li,
+.frameworks li,
+.skills li {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  margin-bottom: 18px;
+}
+
+.languages h4 {
+  display: list-item;
+  list-style-image: url(/assets/Rectangle\ 55.png);
+  list-style-position: inside;
+  margin-top: 136px;
+  margin-bottom: 38px;
+}
+
+.frameworks h4 {
+  display: list-item;
+  list-style-image: url(/assets/Rectangle\ 56.png);
+  list-style-position: inside;
+  margin-bottom: 38px;
+}
+
+.skills h4 {
+  display: list-item;
+  list-style-image: url(/assets/Rectangle\ 57.png);
+  list-style-position: inside;
+  margin-bottom: 38px;
+}
+
+#contact-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 33px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-top: 130px;
+  margin-bottom: 73px;
+  padding-left: 10px;
+  padding-right: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  width: 327px;
+  height: 216px;
+}
+
+.contact input {
+  display: flex;
+  justify-content: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  align-items: center;
+  border-style: none none solid none;
+  border-width: 100%;
+  border-color: white;
+  padding: 10px;
+  width: 100%;
+  margin: 25px 5px;
+}
+
+textarea {
+  width: 100%;
+  background: #fbf8f7;
+  border-style: none;
+  padding: 15px;
+  height: 114px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  margin-bottom: 24px;
+}
+
+.get-intouch-btn {
+  padding: 12px;
+  width: auto;
+  height: 48px;
+  background: #ff6b00;
+  border: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  margin: auto auto 124px auto;
+}
+
+.end-border {
+  border-color: #dfe1e6;
+}
+
+footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 80px;
+}
+
+.color-primary {
+  color: #fff;
+}
+
```

Comment:
- [ ] [OPTIONAL ]Because the previous reviewer did not mention it, this comment will be marked as optional, but your MIT license link is broken, I would suggest you work to fix it for the best of your website and your professionalism.

<img width="698" alt="1" src="https://user-images.githubusercontent.com/77843050/212499589-d29dda3e-13d9-43be-8ca7-a720bc392296.png">

Reply:
@joskalenda ok, thanks